app/lib/widgets/ChatListItem.dart

```diff
@@ -99,91 +99,82 @@ class _ChatListItemState extends State<ChatListItem> {
 
   Widget buildSubtitle(BuildContext context) {
     if (widget.latestMessage == null) {
-      return widget.typingUsers.isEmpty
-          ? const SizedBox()
-          : Text(
-              _multiUserTextBuilder(widget.typingUsers),
-              style: ChatTheme01.latestChatStyle.copyWith(
-                fontStyle: FontStyle.italic,
-              ),
-            );
+      if (widget.typingUsers.isEmpty) {
+        return const SizedBox();
+      }
+      return Text(
```

Comment:
Won't this run regardless of ```if``` above is true or false? I suppose the conditional operator removed works like ```if```` ````else```.